manifests/init.pp

```diff
@@ -440,19 +472,21 @@
       group   => $service_config_dir_group,
       mode    => $service_config_perms,
       content => template($template_snmptrapd),
-      require => Package['snmpd'],
+      require => $manage_snmp_conf_requires,
     }
   }
 
-  file { 'snmpd.sysconfig':
-    ensure  => $file_ensure,
-    path    => $sysconfig,
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0644',
-    content => template($template_snmpd_sysconfig),
-    require => Package['snmpd'],
-    notify  => Service['snmpd'],
+  unless $facts['os']['family'] in ['FreeBSD', 'Darwin'] {
```

Comment:
nit: if not is easier to understand than unless

Reply:
I disagree here, "if !" seems less succinct than "unless", as the "!" is often overlooked when scanning through code. That said, if this is what is blocking the merge 6 months on, I'm happy to change it.